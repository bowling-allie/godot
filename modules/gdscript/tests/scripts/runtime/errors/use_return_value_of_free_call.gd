#debug-only: Runtime checks for using the result of a void method is disabled in release

func test():
	var obj
	obj = Node.new()
	print(obj.free())
