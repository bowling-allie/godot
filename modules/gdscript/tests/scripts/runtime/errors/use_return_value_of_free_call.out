GDTEST_RUNTIME_ERROR
~~ WARNING at line 6: (UNSAFE_METHOD_ACCESS) The method "free()" is not present on the inferred type "Variant" (but may be present on a subtype).
>> SCRIPT ERROR at runtime/errors/use_return_value_of_free_call.gd:6 on test(): Trying to get a return value of a method that returns "void"
